src/main/java/ome/security/basic/BasicSecuritySystem.java

```diff
@@ -553,7 +558,8 @@ public void loadEventContext(boolean isReadOnly, boolean isClose) {
         }
 
         final ome.model.meta.Session session;
-        if (isReadOnly) {
+        if (isShare) {
```

Comment:
I'm unsure where `isReadOnly` has gone.

Reply:
Following from https://github.com/openmicroscopy/omero_server_private/pull/5/files#r363291960 I found that it probably is unnecessary for read-only.

Reply:
After a chat, I now understand that this refers to my previous comment that `isReadOnly` was likely an unneeded performance optimization and can be removed.